src/helpers/attrValidation.js

```diff
@@ -202,3 +202,12 @@ export const linkValidate = memoize((langCode, required = false) => input => (
 export const listRequired = value => (
   !value || !value.length ? <FormattedMessage id="validateForm.required" /> : undefined
 );
+
+export const imageValidate = memoize((langCode, required = false) => input => (
+  (required && !input && !input.values
```

Comment:
Seems odd to me. If `input` is falsy, then does that mean `input.values` would throw an error? Also, the only condition that would allow the subsequent checks to be evaluated is for `input` to be falsy. Otherwise, it shouldn't work as expected.

Reply:
hi @raxkaynan ! I've simplified the logic, the problem is that the Image Field has inner fields bound to it, and the same validation function is being used to the inner fields as well.. so I had to create an awkward logic to make it work.. but now, I have break it down into different blocks, and now the issues seems to be resolved.